crates/bin/pcli/src/command/tx.rs

```diff
@@ -1177,9 +1177,9 @@ impl TxCmd {
                         .set_gas_prices(gas_prices)
                         .set_fee_tier((*fee_tier).into());
 
-                    for position_id in positions_to_close_now {
+                    for (id, _) in positions_to_close_now {
```

Comment:
Nit: would prefer to keep `position_id` var name for comprehension. Also makes the diff smaller and easier to review.